### PR TITLE
feat: style sign-in-page

### PR DESCRIPTION
**Pull request by rafaelramalho19**
- Styles sign-in-page
- Adds page background variant possibility

## ⚠️ TODO IN OTHER PRs: 

- Fix navbar background (@jdiogopeixoto)
- Fix footer positioning

## 👀  Visuals 
image

### CODE REVIEW

**Comment by alanshaw** [comment]
> Adds page background variant possibility

I'm not sure I understand this bit!

The background looks much darker than the design - I'm not sure it's on purpose?

Screenshot 2021-07-16 at 21 22 56

File: packages/website/components/layout.js [hunk 1]
```diff
@@ -32,7 +34,7 @@ export default function Layout({
   const shouldWaitForUser = needsUser && status === 'loading'
 
   return (
-    <div>
+    <div className={clsx(pageBgColor, 'min-h-screen')}>
```
Comment (jdiogopeixoto) [inline packages/website/components/layout.js:37]: We need to add `flex flex-col` here to solve the layout issue

File: packages/website/pages/login.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+ *
+ * @returns {{ props: import('../components/types.js').LayoutProps}}
+ */
 export function getStaticProps() {
   return {
     props: {
       title: 'Login - Web3 Storage',
       redirectTo: '/files',
       redirectIfFound: true,
+      pageBgColor: 'bg-w3storage-red-dark',
```
Comment (jdiogopeixoto) [inline packages/website/pages/login.js:19]: This should be using the color `w3storage-red` according to Figma

**Review by jdiogopeixoto — CHANGES_REQUESTED** [review]
Looks good! Only left 2 small comments

File: packages/website/pages/login.js (hunk 2 above)
Comment (rafaelramalho19) [reply packages/website/pages/login.js:19]: You're right good catch

File: packages/website/components/layout.js (hunk 1 above)
Comment (rafaelramalho19) [reply packages/website/components/layout.js:37]: Was gonna leave that for another task, but I added now

**Review by jdiogopeixoto — APPROVED** [review]
🚀

**Pull request merged**